xsuite/src/world/signer.ts

```diff
@@ -62,24 +63,47 @@ export class Keystore {
     this.password = password;
   }
 
-  static async createFile(filePath: string) {
+  static async createFile(filePath: string, pemPath?: string) {
```

Comment:
A more general way of doing might be better here. `createFile` and `createFile_unsafe` might take an additional and optional argument `data`. And in case `data` is specified, it would be used instead of generating the mnemonic.